src/window_main/components/economy/EconomyRow.tsx

```diff
@@ -171,16 +174,29 @@ function countDupesArray(array: string[]): Record<string, number> {
 
 function CardPoolAddedEconomyValueRecord(props: CardPoolAddedEconomyValueRecordProps) {
   const { addedCardIds, aetherizedCardIds } = props;
-  const addedUniques = (addedCardIds.length > 0) ? countDupesArray(addedCardIds) : undefined;
-  const aetherUniques = (aetherizedCardIds.length > 0) ? countDupesArray(aetherizedCardIds) : undefined;
+
+  return (<>{AddedCardList(addedCardIds)}{AetherizedCardList(aetherizedCardIds)}</>);
+}
+
+function AddedCardList(addedCardIds:string[]){
+  return InventoryCardList(addedCardIds, false);
+}
+
+function AetherizedCardList(aetherizedCardIds:string[]){
+  return InventoryCardList(aetherizedCardIds, false);
+}
+
+function InventoryCardList(cardsList: string[], isAetherized: boolean) {
```

Comment:
Since we're in here, let's make this an actual set of props instead.
```suggestion
interface InventoryCardListProps {
  cardList: string[];
  isAetherized?: boolean;
}

function InventoryCardList(props: InventoryCardListProps ) {
  const { cardList, isAetherized} = props;
```